application.py

```diff
@@ -20,7 +20,14 @@
 import spots_available
 import preregistration
```

Comment:
Can you add some comments on what everything means here?
- preregistration
- catalog_print
- catalog_full
- catalog_full_print
- postregistration
- print_schedule

Actaully it might be good to have comments describing the purpose of all the pages.